CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [6.2.2] - unreleased
 
+- Changed: [[#293](https://github.com/ethereum/evmc/pull/293)]
+  In C++ API `evmc::result::raw()` renamed to `evmc::result::release_raw()`.
```

Comment:
Is this breaking?

Reply:
I think it is, but the comment should at least be merged into 6.2

Reply:
It's breaking, but 6.2 has broken C++ API anyway. Besides, this method is used internally.

Reply:
Just checked, `raw` was introduced in 6.2.1 via #257. I don't think renaming it is fixing a bug, the bug was fixed before the renaming, wasn't it?